PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/subscription/SubscriptionComboBox.java

```diff
@@ -31,6 +31,13 @@
 import java.util.Objects;
 
 public class SubscriptionComboBox extends AzureComboBox<Subscription> {
+
+    public SubscriptionComboBox() {
+        super();
+        final List<Subscription> items = AzureMvpModel.getInstance().getSelectedSubscriptions();
+        this.setValue(items.get(0)); // select the first subscription
```

Comment:
avoid IndexOutOfBoundsException

Reply:
user can not come here if he doesn't have any subscriptions.